plugin/src/main/java/net/dzikoysk/funnyguilds/shared/bukkit/ItemUtils.java

```diff
@@ -119,13 +119,14 @@ public static ItemStack parseItem(String itemString) {
         });
 
         ItemBuilder item = new ItemBuilder(material, amount.orElseGet(1), data.orElseGet(0));
-        FunnyFormatter formatter = new FunnyFormatter().register("_", "").register("{HASH}", "#");
+        FunnyFormatter formatter = new FunnyFormatter().register("_", " ").register("{HASH}", "#");
```

Comment:
Wszystkie zmiany w tym pliku, są całkowicie unrelated do PR i nawet nie mają związku z systemem zaproszeń. 💢

Reply:
Tak, PR z tym zostało przepchnięte ale błąd w tym pliku znalazłem podczas robienia tego PR'a i zapomniałem zrobić nowego brancha 🤦‍♂️

Reply:
W takim razie należy to usunąć z tego PR. 😳

Reply:
Mam zrobić drop z tego commita przez rebase -> forse push?

Reply:
Mergnąłem ci te zmiany z mastera FG i pushnąłem do twojego brancha - teraz już jest git